loops.js

```diff
@@ -17,7 +18,13 @@
 // loop...
 // klaar!
  // ==========================================
-
+for (let i = 0; i < 5; i++) {
+    if (i < 4 ) {
+        console.log("loop")
```

Comment:
Ik heb dit ook precies zo gedaan. In de uitwerkingen is het andersom, maar dit is denk ik ook goed!